### DISCUSSION TITLE
Icon for Neovim in custom?

### DISCUSSION THREAD

**Discussion by georgeguimaraes** [ryanoasis/nerd-fonts#1380] [Ideas]
Hi,

Would you consider adding Neovim icon as custom directly into nerd-fonts?

Reasons:

1. There's no Neovim icon in upstream repos. However, there are some PRs lingering in some upstream repos that appear discontinued:

    - https://github.com/lukas-w/font-logos/pull/93 (font-logos appear to be unmaintained)
    - https://github.com/vorillaz/devicons/issues/151 (vorillaz/devicons appears to be unmainted, last commit 6y ago)

2. Even the "popular, maintained devicons/devicons" (as described in https://github.com/ryanoasis/nerd-fonts/issues/615) does not have a Neovim icon yet. And their last release was 17 months ago. So even if a PR was accepted upstream, there's gonna be a potentially huge delay until nerd-fonts can use that.

3. There's already `custom-vim` in Nerd Fonts (which I'm using as replacement for a neovim logo).

**Comment by Finii** [comment]
I opened an issue from this, to be able to tag it with v3.1 milestone :-D

* https://github.com/ryanoasis/nerd-fonts/issues/1383

**Reply by Finii** [reply]
Neovim user myself ;-) All of Nerd Fonts has been created with nvim...


![signal-2023-10-13-221933_002](https://github.com/ryanoasis/nerd-fonts/assets/16012374/cfd86ace-b43a-4d30-bcfd-1c2117040ddf)

**Reply by georgeguimaraes** [reply]
You need an Elixir sticker too! =)
